js/src/structured-data-blocks/how-to/components/HowTo.js

```diff
@@ -412,10 +430,12 @@ export default class HowTo extends Component {
 							   ( durationHours && durationMinutes ? __( " [and] ", "wordpress-seo" ): "" ) +
 							   ( durationMinutes === 0 ? "" : "m [minutes]" );
```

Comment:
the two lines above have mixed spaces and tabs, thus there's an ESLint warning:
`warning  Mixed spaces and tabs no-mixed-spaces-and-tabs`

I understand a nice alignment helps readability but then we should make a decision, establish a best practice, and tune the linting appropriately. Can be fixed later.